tests/test_async_await.py

```diff
@@ -56,7 +55,7 @@ async def test_pool_context_manager_timeout(pg_params, loop):
     async with aiopg.create_pool(**pg_params, minsize=1,
                                  maxsize=1) as pool:
         cursor_ctx = await pool.cursor()
-        with pytest.raises(psycopg2.ProgrammingError):
+        with pytest.warns(ResourceWarning):
```

Comment:
@asvetlov I'm pretty unsure about this, but it looks like this behaviour has changed because of psycorg update to 2.8+. Do you have any thoughts about this?

Reply:
I don't monitor psycopg2 changelist but this line looks good to me if the test passes.

Reply:
Cool, thanks.

PS I had tried to figure out why it could happened from change log, but failed to do so 😢